app/src/main/java/com/bogdan/codeforceswatcher/LoadUser.kt

```diff
@@ -0,0 +1,89 @@
+package com.bogdan.codeforceswatcher
+
+import android.support.v4.widget.SwipeRefreshLayout
+import android.util.Log
+import com.bogdan.codeforceswatcher.activity.MainActivity
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import java.util.concurrent.CountDownLatch
+
+object LoadUser {
+
+    var textNotification = ""
```

Comment:
Forming `textNotification` here isn't a good practice. You need to return the list of changes from `loadUser` method and then in `NotificationReceiver` form the text itself.